docs/authelia-on-baremetal.md

```diff
@@ -0,0 +1,462 @@
+    # $editor /etc/redis/redis.conf
+    	Change the following to lines;
+    	supervised systemd
+    	requirepass $redis-password
+    # systemctl restart redis
+
+### Testing redis
+    # redis-cli
+    # auth $redis-password
+    # ping
+    # quit
+
+**NOTE** Redis should answer with PONG
+
+### Configure mariadb
+    # mariadb-password=<mariadb-password>
+    # mysql
+    > CREATE DATABASE `authelia`;
+    > GRANT ALL privileges ON `authelia`.* TO 'authelia'@localhost IDENTIFIED BY '$mariadb-password';
+    > exit
+
+### Configure NGINX
+    # unlink /etc/nginx/sites-enabled/default
+    # cd /etc/nginx/sites-available
+    # cat <<EOF > authelia.conf
+    server {
+            listen 80;
+    
+            location / {
+                    include default_headers;
+                    return 301 https://$server_name$request_uri;
+            }
+    }
+    
+    server {
+            server_name authelia.example.com;
+            listen 443 ssl http2;
+            include ssl.conf;
+    
+            location / {
+                    add_header X-Forwarded-Host authelia.example.com;
+                    add_header X-Forwarded-Proto $scheme;
+                    set $upstream_authelia http://authelia.example.com:9091;
+                    proxy_pass $upstream_authelia;
+                    include proxy.conf;
+            }
+    }
+    
+    server {
+            server_name landing.example.com;
+            listen 80;
+            return 301 https://$server_name$request_uri;
+    }
+    
+    server {
+            server_name landing.example.com;
+            listen 443 ssl http2;
+            include ssl.conf;
+            include authelia.conf;
+    
+            location / {
+                    set $upstream_target https://protected.example.com;
+                    proxy_pass $upstream_target;
+                    include auth.conf;
+                    include proxy.conf;
+                    #proxy_ssl_certificate /etc/nginx/ssl/xxx.example.com.cert;
+                    #proxy_ssl_certificate_key /etc/nginx/ssl/xxx.example.com.key;
+            }
+    }
+    EOF
+
+**NOTE** If protected.example.com needs a different SSL certificate than landing.example.com you can un-comment the two proxy_ssl_certificate lines and point them to the correct certificates. It is even possible to have protected.example.com on a completely different domain-name but that is not guaranteed to work without issues.
+
+### Other NGINX files needed
+    # cd /etc/nginx
+    # cat <<EOF > auth.conf
+    # Basic Authelia Config
+    auth_request /authelia;
+    auth_request_set $target_url $scheme://$http_host$request_uri;
+    auth_request_set $user $upstream_http_remote_user;
+    auth_request_set $groups $upstream_http_remote_groups;
+    proxy_set_header X-Forwarded-User $user;
+    proxy_set_header X-Forwarded-Groups $groups;
+    error_page 401 =302 https://authelia.example.com/#/?rd=$target_url;
```

Comment:
I have been testing your doc and i could not get redirection to work.
Looking around I found this: https://github.com/authelia/authelia/issues/466#issuecomment-566786122

```
error_page 401 =302 https://authelia.example.com/?rd=$target_url;
```
After changing my auth.conf line to the above did redirection work.

Still playing with it but just FYI

Regards